engine/src/main/java/org/entando/entando/web/common/model/Filter.java

```diff
@@ -20,8 +20,10 @@
 
 public class Filter {
 
+    private static final String DEFAULT_OP = "like";
```

Comment:
This looks really brittle to me. Why do we have the default as "like"?

Reply:
"like" operator is generally used for a search with filters, when you want more than one specific result, so I think is better than "eq" operator, as default. It's up to managers and daos to ensure that "like" works fine also with numbers and dates.

Reply:
It was "null" before though so it would fail if not specified. This seems like a really big change. @eugeniosant and @stefanopuddu what do you guys think?

Reply:
you still don't need to specify a filter: the default operator is active only if you add a filter in your request. on the other side, if we remove the default operator, it always fails if we add a filter without specifying an operator.

Reply:
Actually the `like` operator was implicit in the previous implementation.
Check the `getFieldSearchFilter()` method. This is where the `Filter` is (must be) converted into a working Entando filter